dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/user/MeController.java

```diff
@@ -265,6 +273,25 @@ public void updateCurrentUser( HttpServletRequest request, HttpServletResponse r
                 conflict( "Invalid format for WhatsApp value '" + user.getWhatsApp() + "'" ) );
         }
 
+        FileResource avatar = currentUser.getAvatar();
+        if ( avatar != null )
+        {
+            FileResource fileResource = fileResourceService.getFileResource( avatar.getUid() );
+            if ( fileResource == null )
+            {
+                throw new WebMessageException(
```

Comment:
Please use `ConflictException` instead. 
Would be nice if you can also remove the `throws WebMessageException` from the method and use the new exceptions where any other `WebMessageException` is thrown.